### ISSUE TITLE
Login redirecting to admin/ instead of LOGIN_REDIRECT_URL

### ISSUE THREAD

**Issue by stefvonb** [AngellusMortis/django_microsoft_auth#449]
* Django Microsoft Authentication Backend version: 2.4.0
* Django version: 3.2
* Python version: 3.9
* Operating System: Windows
* Browser and version: Brave (i.e. Chrome)

### Description

The default javascript will send someone to the admin page if there is no "next" querystring. (login.js - line 113).

What should happen is that it goes to the Django LOGIN_REDIRECT_URL.

### Environment Setup Steps

This is using Django on a Heroku app, it's pretty standard and comes from your test examples.

### Steps to Reproduce

Set up the LOGIN_REDIRECT_URL and then go to the login page without a "next" querystring. It will take you to admin.

**Comment by aydtmiri** [comment]
Were you able to sort it out? I'm facing the same problem atm

**Comment by stefvonb** [comment]
No, I didn't manage to do it properly. It mainly happened when I logged out and in again, so a quick-and-dirty fix was to add this to the settings file:

`LOGOUT_REDIRECT_URL = "/login/?next=/"`

**Comment by AngellusMortis** [comment]
Javascript provided is designed to only function as an example. It is not designed to be used anywhere outside of the admin site. 

If you would like to redirect somewhere else, write your own frontend login code.
